### PR TITLE
[exchange tests] add typing tests

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/ccxt/ccxt/pull/8804 and https://github.com/ccxt/ccxt/pull/8805 in a ccxt version

### CODE REVIEW

File: tests_additional/real_exchanges/real_exchange_tester.py [hunk 1]
```diff
@@ -125,3 +128,19 @@ def get_time():
     @staticmethod
     def ensure_elements_order(elements, sort_key, reverse=False):
         assert sorted(elements, key=lambda x: x[sort_key], reverse=reverse) == elements
+
+    @staticmethod
+    def check_ticker_typing(ticker, check_open=True, check_high=True, check_low=True,
```
Comment (Herklos) [inline tests_additional/real_exchanges/real_exchange_tester.py:133]: :+1:

**Review by Herklos — APPROVED** [review]

File: tests_additional/real_exchanges/test_binance.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
 class TestBinanceRealExchangeTester(RealExchangeTester):
     EXCHANGE_NAME = "binance"
     SYMBOL = "BTC/USDT"
+    SYMBOL_2 = "ETH/BTC"
```
Comment (Herklos) [inline tests_additional/real_exchanges/test_binance.py:33]: 👍

**Review by Herklos — APPROVED** [review]

File: tests/exchanges/traders/test_trader.py [hunk 3]
```diff
@@ -557,7 +557,8 @@ async def test_close_filled_sell_limit_order_one_in_two(self):
         assert second_limit_buy in orders_manager.get_open_orders()
 
         assert initial_portfolio != portfolio_manager.portfolio
-        assert portfolio_manager.portfolio.portfolio["BQX"][commons_constants.PORTFOLIO_AVAILABLE] == 2
+        # 0.002 as fees
+        assert portfolio_manager.portfolio.portfolio["BQX"][commons_constants.PORTFOLIO_AVAILABLE] == 1.998
```
Comment (Herklos) [inline tests/exchanges/traders/test_trader.py:561]: What changed ?

Comment (GuillaumeDSM) [reply tests/exchanges/traders/test_trader.py:561]: the way binance fees are computed in ccxt

Comment (Herklos) [reply tests/exchanges/traders/test_trader.py:561]: I don't understand why this is the only test impacted. If ccxt is now computing fees it's a major change

Comment (GuillaumeDSM) [reply tests/exchanges/traders/test_trader.py:561]: due to this deletion, the way ccxt computes fees for binance has changed https://github.com/ccxt/ccxt/commit/bbf545f84eee84fc930b8c89fe140cac03664531#diff-2fdc97a66b336a92bf28e3ea3665811efbb6c89e3d3c5047ad5265d483188a0bL1063
This is the only test which relies on ccxt fees apparently

File: tests/exchanges/traders/test_trader.py [hunk 4]
```diff
@@ -547,7 +552,11 @@ async def test_close_filled_sell_limit_order_one_in_two(self):
         # Fill only 1st one
         limit_buy.filled_price = 4
         limit_buy.status = OrderStatus.FILLED
-        await limit_buy.on_fill(force_fill=True)
+        with patch.object(ccxt.async_support.binance, "calculate_fee", Mock(return_value=FEES_MOCK)) \
```
Comment (Herklos) [inline tests/exchanges/traders/test_trader.py:555]: :+1:

File: tests/exchanges/traders/test_trader.py [hunk 5]
```diff
@@ -587,7 +596,10 @@ async def test_close_filled_sell_limit_order(self):
 
         assert not trades_manager.trades
 
-        await limit_buy.on_fill(force_fill=True)
+        with patch.object(ccxt.async_support.binance, "calculate_fee", Mock(return_value=FEES_MOCK)) \
```
Comment (Herklos) [inline tests/exchanges/traders/test_trader.py:599]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**